docs/en-us/2.0.3/user_doc/About_DolphinScheduler/About_DolphinScheduler.md

```diff
@@ -1,3 +1,5 @@
+# About DolphinScheduler
```

Comment:
Why we should change this? I found out you change it in your previous PR

Reply:
> Why we should change this? I found out you change it in your previous PR

small mistake @zhongjiajie

Reply:
maybe you can revert commit 2178fd2